tests/user.test.js

```diff
@@ -0,0 +1,205 @@
+const request = require("supertest");
+const jwt = require("jsonwebtoken");
+
+const { createApp } = require("../app");
+const { AppDataSource } = require("../src/models/dataSource");
+
+describe("Update user info", () => {
+  let app;
+  let userId;
+  let accessToken;
+
+  beforeAll(async () => {
+    app = createApp();
+    await AppDataSource.initialize();
+    await AppDataSource.query(
+      `
+      INSERT INTO socials (
+        name
+      ) VALUES ("test")
+      `
+    );
+    await AppDataSource.query(
+      `
+      INSERT INTO gender (
+        name
+        ) VALUES ("male")
+        `
+    );
+    const user = await AppDataSource.query(
+      `
+      INSERT INTO users (
+        email,
+        sns_id,
+        social_id
+      ) VALUES ("test@email.com", 13123214, 1)
+      `
+    );
+
+    await AppDataSource.query(
+      `
+      INSERT INTO users (
+        email,
+        nickname,
+        sns_id,
+        social_id
+      ) VALUES ("test2@email.com", "중복된 닉네임", 13123214, 1)
+      `
+    );
+
+    userId = user.insertId;
+    accessToken = jwt.sign({ id: userId }, process.env.SECRET_KEY, { expiresIn: "1h" });
+  });
+
+  afterAll(async () => {
+    await AppDataSource.query("SET FOREIGN_KEY_CHECKS=0");
+    await AppDataSource.query(`TRUNCATE socials`);
+    await AppDataSource.query(`TRUNCATE gender`);
+    await AppDataSource.query(`TRUNCATE users`);
+    await AppDataSource.query("SET FOREIGN_KEY_CHECKS=1");
```

Comment:
```suggestion
    await AppDataSource.query("SET FOREIGN_KEY_CHECKS=0");
    await AppDataSource.query(`TRUNCATE users`);
    await AppDataSource.query(`TRUNCATE socials`);
    await AppDataSource.query(`TRUNCATE gender`);
    await AppDataSource.query("SET FOREIGN_KEY_CHECKS=1");
```
users가 gender_id, social_id를 가지고 있는 구조 맞을까요?
FK를 가지고 있는 테이블부터 truncate하시는 것이 좋습니다 :)

Reply:
수정했습니다.!!